custom/icds_reports/static/icds_reports/js/spec/adolescent-girls.directive.spec.js

```diff
@@ -15,7 +15,6 @@ describe('Adolescent Girls Directive', function () {
 
     beforeEach(module('icdsApp', function ($provide) {
         utils.provideDefaultConstants($provide, {});
-        $provide.constant("haveAccessToFeatures", false);
```

Comment:
is this removed just because it was not used anywhere or its removed because way of injecting vars changed?

Reply:
this is redundant, as the utils function in the line above it `provideDefaultConstants` add this constant